# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/helvum/hotfix-desktopfiles.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

This command is not required as stated above in meson.build but it is not
conditionalized here for some reason. Disable it and wait for upstream
to fix this.

  - NoTag <notag@t2sde.org>

--- helvum-0.5.1/meson.build.vanilla	2024-05-02 14:12:50.558937807 +0200
+++ helvum-0.5.1/meson.build	2024-05-02 14:15:32.653963692 +0200
@@ -35,5 +35,5 @@
 
 gnome.post_install(
   gtk_update_icon_cache: true,
-  update_desktop_database: true,
+  #update_desktop_database: true,
 )
